autometa/binning/unclustered_recruitment.py

```diff
@@ -572,7 +573,8 @@ def main():
     )
     prev_num_unclustered = bin_df[bin_df.cluster.isnull()].shape[0]
     if not prev_num_unclustered:
-        raise BinningError("No unclustered contigs are available to recruit!")
+        logger.warning("No unclustered contigs are available to recruit!")
```

Comment:
@WiscEvan Adding a test for this case is something small that also fits in with https://github.com/KwanLab/Autometa/pull/101
or @Sidduppal